### DISCUSSION TITLE
Is it possible to disable an autofocus in Modal component?

### DISCUSSION THREAD

**Discussion by Marsony** [mantinedev/mantine#5606] [Q&A]
Hello everyone, thank you for this great library!

We're currently working on a Modal component for mobile devices, using Mantine solely as a compound component.

When we open the modal, Mantine, by default, sets focus on the first interactive element, which in our case is a custom close button.

Is it possible to disable the initial autofocus on the first interactive element? Thank you!

**Comment by rtivital** [comment]
You can disable focus trap with `trapFocus={false}`. Note that it is accessibility violation.

**Reply by Marsony** [reply]
Thank you for your answer! However, I don't need to disable the focus trap. I just want to disable the initial auto-focus.

**Reply by rtivital** [reply]
Then create a hidden element that will receive the initial focus
